lib/sen_tweet_web/telemetry.ex

```diff
@@ -60,6 +60,7 @@ defmodule SenTweetWeb.Telemetry do
       last_value("bitfeels.pipeline.sentiment.number_of_events"),
       counter("bitfeels.pipeline.sentiment.id", tags: [:track]),
       summary("bitfeels.pipeline.sentiment.score", tags: [:track]),
+      sum("bitfeels.pipeline.sentiment.score", tags: [:tweet_type]),
```

Comment:
I discovered that the `summary` metric does not support grouping across metadata. e.g.:

```elixir
summary("bitfeels.pipeline.sentiment.score", tags: [:tweet_type])
```
or
```elixir
summary("bitfeels.pipeline.sentiment.score", tags: [:track, :tweet_type])
```

do not produce the expected behavior. `sum` distributes as expected and is still useful, though I'd rather see the average.

Reply:
Interesting, I didn't notice that before but that seems to be the case. 

They are using a summary with tags in one of the built in metrics 🤔 

```
summary("phoenix.router_dispatch.stop.duration",
        tags: [:route],
        unit: {:native, :millisecond}
      )
```

Fun fact is this library has been updated since I started on this PR. Well update and see if that fixes anything.

Reply:
I found this open issue, https://github.com/phoenixframework/phoenix_live_dashboard/issues/129

Looks like what you found is known issue. It'll get fixed at some point 🤣 

Distribution charts are not fully baked as well.